src/include/binder/binder.h

```diff
@@ -112,14 +113,13 @@ class Binder {
     shared_ptr<NodeExpression> bindQueryNode(const NodePattern& nodePattern, QueryGraph& queryGraph,
         PropertyKeyValCollection& collection);
     shared_ptr<NodeExpression> createQueryNode(const NodePattern& nodePattern);
-    inline unordered_set<table_id_t> bindNodeTableIDs(const vector<string>& tableNames) {
+    inline vector<table_id_t> bindNodeTableIDs(const vector<string>& tableNames) {
         return bindTableIDs(tableNames, NODE);
     }
-    inline unordered_set<table_id_t> bindRelTableIDs(const vector<string>& tableNames) {
+    inline vector<table_id_t> bindRelTableIDs(const vector<string>& tableNames) {
```

Comment:
Why not just return `unordered_set`? same for `bindNodeTableIDs`.

Reply:
We want to ensure the order of table_id to be the same across different platforms.